test/integration_test_helper.rb

```diff
@@ -213,6 +212,7 @@ def database_cleaner_strategy
 
   def login_admin
     SSO.register_method(TestSSO)
+    visit '/'
```

Comment:
while this fixes some tests, some are also failing because of this (e.g. `TEST=test/integration/fact_value_test.rb`)